src/controllers/CheckYourAppealController.ts

```diff
@@ -39,7 +40,7 @@ const navigation = {
 export class CheckYourAppealController extends SafeNavigationBaseController<any> {
     constructor() {
         super(template, navigation, undefined, undefined, [AppealStorageFormActionProcessor,
-            InternalEmailFormActionProcessor, UserEmailFormActionProcessor]);
+            InternalEmailFormActionProcessor, UserEmailFormActionProcessor, SessionCleanupProcessor]);
```

Comment:
why are we setting
`appData.submittedAppeal = appData.appeal`
at the check-your-appeal page? An appeal has not been submitted at this point

Reply:
At the point of execution of the SessionCleanupProcessor, which is after the appeal has been submitted by the AppealStorageFormActionProcessor, we can safely say that the appeal has been submitted to mongo and therefore we can mark this appeal as 'complete' by moving it to the `submittedAppeal` field. And deleting the current `appeal` from the session

Reply:
ah my bad, my head was in middleware mode, the processor makes more sense

Reply:
Let's say the customer has submitted an appeal, and they navigate back to the start of the journey, will their stale appeal data from their active session populate the fields on the `appeal-a-penalty/penalty-reference` page? Or does this get cleared before they get to the summary page?